src/components/new-request/search.vue

```diff
@@ -165,7 +165,7 @@
           <v-col v-if="isXproFlow && isFederal" cols="12" :md="isSelectedCompanyXPro ? 10 : 8">
             <ul class="bullet-points">
               <li>Federally incorporated businesses do not need a Name Request.</li>
-              <li v-if="isRestoration && isSelectedXproAndRestorable">
+              <li v-if="isRestoration && isSelectedXproAndRestorable" class="mb-n10">
```

Comment:
Instead of pulling up the bottom of this block, could you remove the top margin of whatever component is below this?

Reply:
Much nicer!

What I have found is that it's easier to lay things out if you choose all-top-whitespace or all-bottom-whitespace. When there's a mix, it gets complicated fast!

Reply:
I totally agree Sev.